# ID ящика, на который направлены данные
int16 can_id

# Массив направляемых данных
int8[8] can_data